### PR TITLE
Replace `kubectl` with `oc`

### PR DESCRIPTION
**Pull request by xenolinux**
Tracking changes from this [PR](https://github.com/openshift/openshift-docs/pull/54122) 

Version(s): 4.12

Issue: None

Link to docs preview: https://54637--docspreview.netlify.app/openshift-enterprise/latest/networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/creating-instance-aws-load-balancer-controller.adoc [hunk 1]
```diff
@@ -139,7 +141,7 @@ spec:
 +
 [source,terminal]
 ----
-$ HOST=$(kubectl get ingress -n echoserver echoserver -o json | jq -r '.status.loadBalancer.ingress[0].hostname')
+$ HOST=$(oc get ingress -n echoserver echoserver --template='{{index .status.loadBalancer.ingress 0}}{{.hostname}}')
```
Comment (lihongan) [inline modules/creating-instance-aws-load-balancer-controller.adoc:144]: The original `-o json | jq ...` is correct, if we want a replacement with `--template`, the it should be like `--template='{{(index .status.loadBalancer.ingress 0).hostname}}'`

Comment (xenolinux) [reply modules/creating-instance-aws-load-balancer-controller.adoc]: Addressed

**Comment by lihongan** [comment]
/lgtm
Thank you!

**Review by jldohmann — APPROVED** [review]
lgtm!

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Review by skrthomas — COMMENTED** [review]
LGTM. Ok to merge post freeze.

**Pull request merged**